CMakeLists.txt

```diff
@@ -165,10 +165,28 @@ if(ASPECT_WITH_WORLD_BUILDER)
   MESSAGE(STATUS "Using World Builder version ${WORLD_BUILDER_VERSION} found at ${WORLD_BUILDER_SOURCE_DIR}.")
 
   IF(WORLD_BUILDER_VERSION VERSION_GREATER_EQUAL 0.6.0)
-    ADD_SUBDIRECTORY("${WORLD_BUILDER_SOURCE_DIR}" ${CMAKE_BINARY_DIR}/world_builder/)
-    INCLUDE_DIRECTORIES("${CMAKE_BINARY_DIR}/world_builder/include/")
-    # if we configured with 0.5.0 before, we have a stray include file here. delete it.
-    FILE(REMOVE "${CMAKE_BINARY_DIR}/include/world_builder/config.h")
+    IF(${CMAKE_BUILD_TYPE} MATCHES "DebugRelease")
+      SET(WB_ENABLE_TESTS OFF)
+      SET(WB_ENABLE_APPS OFF)
+      SET(WB_ENABLE_HELPER_TARGETS OFF)
+
+      SET(CMAKE_BUILD_TYPE Debug)
+      SET(WB_TARGET "WorldBuilderDebug")
+      SET(CMAKE_CXX_FLAGS "-g")
+      ADD_SUBDIRECTORY("${WORLD_BUILDER_SOURCE_DIR}" ${CMAKE_BINARY_DIR}/world_builder/)
+      # if we configured with 0.5.0 before, we have a stray include file here. delete it.
+      FILE(REMOVE "${CMAKE_BINARY_DIR}/include/world_builder/config.h")
+
+      SET(CMAKE_BUILD_TYPE Release)
+      SET(CMAKE_CXX_FLAGS "-O3 -DNDEBUG")
+      ADD_SUBDIRECTORY("${WORLD_BUILDER_SOURCE_DIR}" ${CMAKE_BINARY_DIR}/world_builder_release/)
+      # if we configured with 0.5.0 before, we have a stray include file here. delete it.
+      FILE(REMOVE "${CMAKE_BINARY_DIR}/include/world_builder_release/config.h")
```

Comment:
Not sure this line is needed here again, it is already in line 177/178 and I dont think 0.5.0 would be configured so that it creates this file in world_builder_release (see line 197 which would create the file). But these two lines are needed in the else branch down before line 188.

Reply:
Makes sense. I decided to move it outside the if/else to avoid duplicating the code. OK?

Reply:
Yes that makes even more sense. Good to go :+1: